modules/mongodb/src/main/java/org/testcontainers/containers/MongoDBAtlasLocalContainer.java

```diff
@@ -0,0 +1,43 @@
+package org.testcontainers.containers;
```

Comment:
```suggestion
package org.testcontainers.mongodb;
```

Reply:
@eddumelendez I'm concerned that this would be a breaking change for existing users of the MongoDBContainer (assuming we would move both classes to this new package). Perhaps we should leave both in the current package unless making a breaking change to move both to the new package?

Reply:
Let's only move `MongoDBAtlasLocalContainer`, since it is a new implementation there is no issues.

Reply:
I tried doing that, but the container uses a custom ContainerDef extension to allow it to override the wait strategy. This class is package local, so requires that we remain in the org.testcontainers.containers package. 

The use of a custom container def will be important for future changes too, since we will want to add custom with... statements for atlas search indexes and seed data once the container supports that (assuming MongoDB agree and add the necessary dependencies to the container).